src/components/ImageSlider/Arrow.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+class Arrow extends Component {
+  render() {
+    const {
+      name,
+      color,
+      enableBackground,
+      backgroundColor,
+      backgroundRounding,
+      enableBorder,
+      borderSize,
+      borderColor,
+      size,
+      onPress,
+      style,
+      disabled,
+    } = this.props
+
+    const viewHeight =
+      size + (enableBackground && enableBorder ? borderSize : 0)
+
+    const viewStyle = {
+      backgroundColor: enableBackground ? backgroundColor : null,
+      borderRadius: enableBackground ? backgroundRounding : null,
+      borderColor: enableBorder && enableBackground ? borderColor : null,
+      borderWidth: enableBorder && enableBackground ? borderSize : null,
+      height: viewHeight,
+      width: viewHeight,
+      display: 'flex',
+      alignSelf: 'center',
+      alignItems: 'center',
+      justifyContent: 'center',
```

Comment:
All of these static properties should be in a stylesheet. You'll need to switch to the preferred `style={[styles1, styles2, ...]}` syntax though to do this.